README.md

```diff
@@ -4,24 +4,38 @@ Logz.io provides an automated deployment process to simplify the process of ship
 This integration forwards logs from your Azure Blob Storage	
 to your Logz.io account.
 
+![Integration-architecture](images/logzio-blob-Diagram.png)
+
+
 <!-- tabContainer:start -->
 <div class="branching-container">
 
+## Deployment options:
+
 * [Use your existing blob storage account](#existing-blob-config)
 * [Create a new blob storage account](#new-blob-config)
 
+These deployments will create the following services:
+* Serveless Function App
+* Application Insights
+* App Service Plan
+* Event Hubs Namspace
+* Event Grid System Topic
+* Function's logs Storage Account
+* Blob Storage Account (will be created only if you choose the second deploy option)
+
 <!-- tab:start -->
 <div id="existing-blob-config">
 
-## Use your existing blob storage account
+# Use your existing blob storage account
 
 **Before you begin, you'll need**:
 a blob storage account of the type **StorageV2 (general purpose v2)** . If your existing blob storage account is of any other kind, it will NOT work. Instead, follow the process to set up a new blob storage account.
```

Comment:
```suggestion
A Blob storage account of the type **StorageV2 (general purpose v2)** . If your existing blob storage account is of any other kind, it will NOT work. Instead, follow the process to set up a new blob storage account.
```